lib/perl/Genome/Site/TGI/Extension/UnarchiveInstrumentData.pm

```diff
@@ -0,0 +1,196 @@
+package Genome::Config::AnalysisProject::Command::UnarchiveInstrumentData;
+
+use strict;
+use warnings;
+
+use Genome;
+use Genome::Config::AnalysisProject::Command::UnarchiveInstrumentData; #load real module first
+
+use File::Basename qw();
+use File::Spec qw();
+
+use Sub::Install;
+
+Sub::Install::reinstall_sub ({
+    as => 'unarchive_additional_data',
+    code => \&unarchive_lims_data,
+});
+
+__PACKAGE__->__meta__->property('volume')->is_optional(0);
+
+sub unarchive_lims_data {
+    my $self = shift;
+    my $instrument_data = shift;
+
+    my $lims_unarchive_script_path = $self->resolve_lims_unarchive_script_path();
+
+    my $dv = $self->volume;
+    if ($instrument_data->class ne 'Genome::InstrumentData::Solexa') { return 1; }
+
+    my $bam_path = $instrument_data->bam_path;
+    if (-e $bam_path) { return 1; }
```

Comment:
@tmooney I thought I made a commit to address data that has an archive_path instead of a bam_path?  Are we missing a commit, or did this never get pushed to your branch before the PR?